src/Model/ModelManager.php

```diff
@@ -519,7 +519,9 @@ public function getPaginationParameters(DatagridInterface $datagrid, $page)
     {
         $values = $datagrid->getValues();
 
-        $values['_sort_by'] = $values['_sort_by']->getName();
+        if (isset($values['_sort_by'])) {
```

Comment:
No `instanceof` check?

Reply:
Shouldn't the check be somewhere else ?

if set, $values['_sort_by'] should be a FieldDescriptionInterface.
So I don't expect a check here, I expect an exception when I tried to set something else.

Reply:
It's a public method that uses an untyped array value of the object, so you should check it.

Reply:
@core23 Done